upstream: no
reason: Our SDL2-devel package has a for cross broken sdl2-config.cmake,
this patches switched back to a find module cmake script

---
--- a/CMakeLists.txt	2020-07-19 19:54:00.744509368 +0200
+++ b/CMakeLists.txt	2020-07-19 19:54:00.744509368 +0200
@@ -110,7 +110,7 @@
 find_package(PNG REQUIRED)
 
 find_package(Threads REQUIRED)
-find_package(SDL2 CONFIG)
+find_package(SDL2 REQUIRED MODULE)
 if(SDL2_FOUND)
   message(STATUS "Found SDL2 configuration file")
   if(TARGET SDL2::SDL2)
